liquidations/aave-v2/index.ts

```diff
@@ -1,99 +1,102 @@
-import { gql, request } from "graphql-request"
-import { getPagedGql } from "../utils/gql"
+import { gql, request } from "graphql-request";
+import { getPagedGql } from "../utils/gql";
 
 const query = gql`
-query users($lastId: String) {
-    users(first: 1000, where:{
-        borrowedReservesCount_gt: 0,
-        id_gt: $lastId
-      }) {
-        id
-        reserves{
-          usageAsCollateralEnabledOnUser
-          reserve{
-            symbol
-            usageAsCollateralEnabled
-            underlyingAsset
-            price {
-                priceInEth
-            }
-            decimals
-            reserveLiquidationThreshold
+  query users($lastId: String) {
+    users(first: 1000, where: { borrowedReservesCount_gt: 0, id_gt: $lastId }) {
+      id
+      reserves {
+        usageAsCollateralEnabledOnUser
+        reserve {
+          symbol
+          usageAsCollateralEnabled
+          underlyingAsset
+          price {
+            priceInEth
           }
-          currentATokenBalance
-          currentTotalDebt
+          decimals
+          reserveLiquidationThreshold
         }
+        currentATokenBalance
+        currentTotalDebt
+      }
     }
-    _meta{
-        block {
-            number
-        }
-    }
-}
-`
+  }
+`;
 
 const ethPriceQuery = gql`
-{
-    priceOracleAsset(id: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"){
-        priceInEth
+  {
+    priceOracleAsset(id: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48") {
+      priceInEth
     }
-}
-`
+  }
+`;
 
-const subgraphUrl = "https://api.thegraph.com/subgraphs/name/aave/protocol-v2"
+const subgraphUrl = "https://api.thegraph.com/subgraphs/name/aave/protocol-v2";
+
+const positions = async () => {
```

Comment:
has anything been changed here? if no change has been made better not to change the code since its a pain to review

Reply:
yeah np, if reviewing i prefer to make a PR with changes, and then after its merged make another PR running prettier, which can be quickly merged

but tbh it doesnt really matter